pyproj/enums.py

```diff
@@ -35,14 +35,14 @@ class ProjVersion(IntEnum):
     PROJ_5 = 5
 
 
-class TransformDirection(IntEnum):
+class TransformDirection(Enum):
     """
     Supported transform directions
     """
 
     #: Forward direction
-    FORWARD = 1
+    FORWARD = "FORWARD"
```

Comment:
Small remark: we could also opt for lower case? (that might be a little bit nicer in user code (`transform(..., direction='inverse')` vs `transform(..., direction='INVERSE')`)

Just a remark, no strong preference either way!

Reply:
I thought about doing that, and decided that the upper case version is consistent with the upper case WKT enum strings.

Reply:
Yep, on the other hand, WKT is an existing abbreviation that is written in upper case, while forward/inverse are normal words.

Reply:
That's true. What would be nice is a case insensitive solution.

Reply:
That would preserve past behavior as that's how I had it before.